bmm/src/main/java/org/openehr/bmm/v2/persistence/PBmmSinglePropertyOpen.java

```diff
@@ -31,14 +38,29 @@ public void setType(String type) {
     @JsonIgnore
     @Override
     public PBmmOpenType getTypeRef() {
-        if(serializedTypeRef != null) {
+        if (serializedTypeRef != null) {
             return serializedTypeRef;
         } else if(getTypeDef() == null) {
             return new PBmmOpenType(type);
         }
         return getTypeDef();
     }
 
+    @Override
+    public BmmUnitaryProperty createBmmProperty(BmmClassProcessor processor, BmmClass bmmClass) {
+        PBmmOpenType typeRef = getTypeRef();
+        if (typeRef != null) {
+            BmmParameterType bmmType = typeRef.createBmmType(processor, bmmClass);
+            if (bmmType != null) {
+                BmmUnitaryProperty bmmProperty = new BmmUnitaryProperty(getName(), bmmType, getDocumentation(), nullToFalse(isMandatory()), nullToFalse(isComputed()));
+                populateImBooleans(bmmProperty);
+                return bmmProperty;
+            }
```

Comment:
`createBmmType` always returns a non-null value, so the if-statement can be removed.